cookbooks/mars_mola_topography/doc/mars_topo.md

```diff
@@ -0,0 +1,23 @@
+# Mars topography
+
+*This section was contributed by Cedric Thieulot and Bart Root.*
+
+From the original MOLA data (<https://attic.gsfc.nasa.gov/mola/>), we have created a topography file
+(present in the folder of this cookbook) with a 1 degree resolution in both longitude and latitude
```

Comment:
It isn't actually in the directory. You probably want to add it there.

Reply:
sorry about that !